lib/apiservers/engine/backends/container.go

```diff
@@ -1258,7 +1258,7 @@ func attachStreams(ctx context.Context, vc *viccontainer.VicContainer, clStdin i
 
 	// For stdin, we only have a timeout for connection.  There can be a long duration before
 	// the first entry so there is no timeout for response.
-	plClient, transport := createNewAttachClientWithTimeouts(attachConnectTimeout, 0, 0)
+	plClient, transport := createNewAttachClientWithTimeouts(attachConnectTimeout, 0, attachAttemptTimeout)
```

Comment:
I'll do it post-TP3.  Let's not delay this tag.  It's on my list of post-TP3 clean up and refactoring to do.